homeassistant/components/tuya/select.py

```diff
@@ -218,6 +218,26 @@
             device_class=DEVICE_CLASS_TUYA_FINGERBOT_MODE,
             entity_category=ENTITY_CATEGORY_CONFIG,
         ),
+    ),    # Robot Vacuum
+    # https://developer.tuya.com/en/docs/iot/fsd?id=K9gf487ck1tlo
+    "sd": (
+        SelectEntityDescription(
+            key=DPCode.SUCTION,
+            name="Suction",
+            entity_category=ENTITY_CATEGORY_CONFIG,
```

Comment:
The Tuya integration has moved to use StrEnums for this.

```suggestion
            entity_category=EntityCategory.CONFIG,
```